docs/guides/compose.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Compose
+---
+
+The Tailcall CLI's `compose` command facilitates the merging of configuration files and their conversion between different formats using the `--format` option.
+
+It supports `gql` or `graphql`, `yml` or `yaml`, `json`. Default is `json`.
+
+## Converting Formats with `compose --format`
+
+To convert files between different formats, use the following syntax:
+
+```bash
+tailcall compose <input_files> <output_file> --format format_type
+```
+
+## Example Usage
```

Comment:
Should link to `operators` as the natural next step to learn more about how to write the configuration.